### PR TITLE
[ticket's number]: add login

### PR DESCRIPTION
**Pull request by supparatleelee**

### CODE REVIEW

**Comment by rachata-d** [comment]
Ready

File: src/Context.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  const [name1, setName1] = useState({ name: 'Aom' });
+
+  return (
+    <TestContext.Provider value={{ name, name1 }}>
+      {children}
+    </TestContext.Provider>
+  );
+}
+
+// export { testContext };
```
Comment (supparatleelee) [inline src/Context.js:16]: please remove this comment

File: src/index.js [hunk 2]
```diff
@@ -3,11 +3,14 @@ import ReactDOM from 'react-dom/client';
...
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import TestContextProvider from './Context';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <TestContextProvider>
+      <App />
```
Comment (PAN-WICHAYA) [inline src/index.js:12]: Very good

File: src/Context.js (hunk 1 above)
Comment (rachata-d) [inline src/Context.js:16]: Need some urgent cleaning ***

File: src/Context.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+// useHook
+export const useTestContext = () => {
+  return useContext(TestContext);
```
Comment (supparatleelee) [inline src/Context.js:20]:
```suggestion
  return useContext(Test);
```

**Review by rachata-d — APPROVED** [review]

**Review by PAN-WICHAYA — APPROVED** [review]

**Pull request merged**